deps/wazuh_testing/wazuh_testing/modules/aws/data_generator.py

```diff
@@ -1,11 +1,20 @@
 """Utils to generate sample data to AWS"""
+import csv
+import json
 from datetime import datetime
+from io import StringIO
 from os.path import join
 from uuid import uuid4
 
+from wazuh_testing.tools.utils import get_random_ip, get_random_port, get_random_string
+
 from . import constants as cons
 
 
+def get_random_interface_id():
```

Comment:
```suggestion
def get_random_interface_id() -> str:
```
Also, a docstring should be included